app/util/confluence/index-snapshot.sh

```diff
@@ -17,7 +17,8 @@ FAIL_FAST_ATTEMPTS=15
 
 
 while [ ${COUNTER} -lt ${ATTEMPTS} ];do
-    if sudo su -c "test -f ${SNAPSHOT}"; then
+    size=`sudo su -c "ls -la ${SNAPSHOT}" | awk '{print $5}'`
```

Comment:
Let's use the same approach for getting file size
SNAPSHOT_SIZE=`sudo su -c "du -s ${SNAPSHOT}" | cut -f1"` - size in KB
so in this case would be
MIN_SNAPSHOT_SIZE=5242880  # 5*1024*1024